image-layout.md

```diff
@@ -5,7 +5,7 @@ This layout MAY be used in a variety of different transport mechanisms: archive
 
 Given an image layout and a ref, a tool can create an [OCI Runtime Specification bundle](https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/bundle.md) by:
 
-* Following the ref to find a [manifest](manifest.md#image-manifest), possibly via a [manifest list](manifest-list.md#manifest-list)
+* Following the ref to find a [manifest](manifest.md#image-manifest), possibly via a [image index](image-index.md#manifest-list)
```

Comment:
> The image layout is NOT directory storage.

Image-layout is not *storing* directories, it is [*using* a directory tree to store blobs and references][1].  You can wrap that image-layout tree up in a [tarball (or whatever you want)][2], but it's still a directory-based format.  That format stores [opaque][3], [content-addressable][4] blobs, and [generic references to them][5], so I don't see anything that is specific to images besides some [informative][6] [hints][7].

[1]: https://github.com/opencontainers/image-spec/blob/v1.0.0-rc4/image-layout.md#content
[2]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc4/image-layout.md#L4
[3]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc4/image-layout.md#L17
[4]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc4/image-layout.md#L3
[5]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc4/image-layout.md#L60
[6]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc4/image-layout.md#L19
[7]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc4/image-layout.md#L65